docs/README.md

```diff
@@ -43,16 +31,18 @@ The Nix community is nice. Be nice! Typically this means:
 
 Nothing is set in stone. Think of these as the guidelines for our interactions.
 
-The maintainer of a project gets admin access to the project they maintain. The maintainer is allowed to invite more people into the org and into their team.
+The maintainer of a project gets admin access to the project they maintain. The maintainer can request that more people be invited into the org so they can join their projects team.
 
-If you see a project that is under-maintained, submit PRs to fix it as if it was your own project. If the PRs don't get merged, ping the [org owners](#org-owners)
+If you see a project that is under-maintained, submit PRs to fix it as if it was your own project. If the PRs don't get merged, ping the [nix-community administrators](#nix-community-administrators).
 
-### Org owners
+## nix-community administrators
 
-If you need to contact the org owners, see [the list of org owners](https://github.com/orgs/nix-community/people?utf8=%E2%9C%93&query=+role%3Aowner).
+If you need to contact the org admins, they are listed [here]https://github.com/nix-community/infra#nix-community-administrators).
 
 Come chat with us in [#nix-community](https://matrix.to/#/#nix-community:nixos.org) using the matrix protocol!
 
 ## Sponsors
 
-<img src="https://raw.githubusercontent.com/cachix/cachix/master/logo.png" width="200" alt="Cachix"> provides us with a free 1TB binary cache!
+<img src="https://raw.githubusercontent.com/cachix/docs.cachix.org/master/source/logo.png" width="200" alt="Cachix">
+
```

Comment:
Should we list the people/companies that provide the hosts here?

Reply:
yeah, I was thinking about that today actually.